packages/access-client/src/types.ts

```diff
@@ -90,6 +91,7 @@ export interface Service {
   access: {
     // returns a URL string for tests or nothing in other envs
     authorize: ServiceMethod<AccessAuthorize, string | undefined, Failure>
+    delegate: ServiceMethod<AccessDelegate, void, Failure>
```

Comment:
afaict there is no response type [in the spec](https://github.com/web3-storage/specs/blob/main/w3-access.md#accessdelegate).

It also looks like `void` is a fine `Success` placeholder for now, because I don't expect the invoker to need anything from  the `Success` other than knowing it wasn't a Failure. i.e. if `access/delegate` invocation was handled without error, then the delegations were persisted.